.github/workflows/python-app.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         mypy usl_pipeline/study_area_uploader
 
   cloud_functions:
-    name: Cloud Funcations CI
+    name: Cloud Functions CI
```

Comment:
This name is used in github config. You'll notice the CI for this isn't actually running because the name has changed and the config for the repo is apparently referring to the mispelled name.

It's currently

> Cloud Funcations CI Expected — Waiting for status to be reported

indefinitely. I suspect this PR isn't actually working and we'd need to pull in C & Fortran compilers to build the wheel.

Reply:
Reverted the name and seems to run fine. The previous name change triggers a job but sets the old one to pending: https://github.com/UrbanSystemsLab/climateiq-cnn/actions/runs/9846444330/job/27184227122

lmk what you think!

Reply:
Does look like it builds! The image must already have the compilers & such we need.

Reply:
we could fix the 'Funcations' name in a follow up PR and adjust the GitHub settings to match.